src/main/scala/com/eclipsesource/schema/urlhandlers/ClasspathUrlProtocolHandler.scala

```diff
@@ -0,0 +1,5 @@
+package com.eclipsesource.schema.urlhandlers
+
+case class ClasspathUrlProtocolHandler() extends ClasspathHandler with UrlProtocolHandler {
```

Comment:
Is there a reason for this to be a case class?

- The above `ClasspathHandler` is not a case class
- Parameter-less case classes should really be case objects